build/commonTest.props

```diff
@@ -25,8 +25,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
     <NoWarn>$(NoWarn);SYSLIB0050</NoWarn>
     <NoWarn>$(NoWarn);SYSLIB0051</NoWarn>
+    <NoWarn>$(NoWarn);NU1701</NoWarn>
```

Comment:
Supresses the "_This package may not be fully compatible with your project_" warning. The cause is that `Microsoft.Extensions.TimeProvider.Testing` package is not supported by .NET FWK 4.6.1.